django/db/models/functions/base.py

```diff
@@ -203,8 +203,7 @@ def __init__(self, expression, substring, **extra):
         """
         if not hasattr(substring, 'resolve_expression'):
             substring = Value(substring)
```

Comment:
Setting `_output_field` broke tests because of import issues leading to:

```
django.core.exceptions.ImproperlyConfigured: Requested setting DEFAULT_INDEX_TABLESPACE, but settings are not configured. You must either define the environment variable DJANGO_SETTINGS_MODULE or call settings.configure() before accessing settings.
```

Leaving it as it was with `output_field` argument